css/mistica-common.css

```diff
@@ -1267,3 +1267,78 @@ button.mistica-display-card:has(.mistica-card__media):active:after {
 .mistica-checkbox-label:has(.mistica-checkbox:disabled) > .mistica-checkbox {
     opacity: initial; /* avoid applying 0.5 opacity twice */
 }
+
+/* Counter */
+
+.mistica-counter {
+    border-radius: var(--mistica-border-radius-button);
+    background: var(--mistica-color-backgroundContainer);
+    display: inline-flex;
+    gap: 8px;
+    overflow: hidden;
+    border: 1px solid var(--mistica-color-inputBorder);
```

Comment:
In inverse context the border should be white also

<img width="349" alt="Screenshot 2025-05-16 at 4 44 02 PM" src="https://github.com/user-attachments/assets/7b12c194-65e2-443f-b8aa-4023948dcd20" />